src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrise = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrise - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return basePrise - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The current logic applies only one discount based on the rental period. If the intention is to apply both discounts cumulatively for rentals longer than 7 days, you need to adjust the logic to account for this. Otherwise, the current implementation only applies the long-term discount for rentals of 7 days or more, and the short-term discount for rentals of 3 to 6 days.